xml/System.IO.Compression/BrotliStream.xml

```diff
@@ -481,10 +494,18 @@
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" Index="0" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="cancellationToken">The token to monitor for cancellation requests. The default value is <see cref="P:System.Threading.CancellationToken.None" />.</param>
```

Comment:
Similarly here, for FlushAsync, since compression/decompression here, its worth noting that FlushAsync doesn't do much on decompression and is only relevant for compression.

https://github.com/dotnet/corefx/blob/306fe9d60cc6db02a69d8ea6b88dde2b98fe8a7c/src/System.IO.Compression.Brotli/src/System/IO/Compression/enc/BrotliStream.Compress.cs#L142